package.json

```diff
@@ -11,10 +11,12 @@
   "author": "Sam Hardacre <me@nocturnalmonkey.com>",
   "license": "MIT",
   "dependencies": {
-    "oleg-starter-pack": "^1.0.3"
+    "node": "^12.4.0",
```

Comment:
This can be uninstalled. Definitely my fault since I didn't explicitly say to update `node-sass` when you were trying to fix your failing build.